lib/parser.rb

```diff
@@ -23,16 +23,20 @@ def command_type
       :C_PUSH
     elsif @command.start_with?("pop")
       :C_POP
+    elsif @command.start_with?("label")
+      :C_LABEL
+    elsif @command.start_with?("if-goto")
+      :C_IF
```

Comment:
Refactoring question: is there any way we could make this a `case` instead of repeating `elsif @command.start_with?(…)`?

Reply:
What do you think of this change [`f10925f` (#14)](https://github.com/wildmaples/jack-virtual-machine/pull/14/commits/f10925f2c0384ff519b73a38f563f546d75733b5)?

Reply:
I like it. I left a comment to show an alternative approach but I think yours is good too.